build -publish warning

I am getting these warnings when running `build -publish` on Windows:

```
dotnet pack -nr:false  --configuration Debug
MSBuild version 17.7.1+971bf70db for .NET
C:\Users\bernard\source\repos\icerpc-csharp\build\Common.props(4,5): warning MSB4011: "C:\Users\bernard\source\repos\icerpc-csharp\build\IceRpc.Version.props" cannot be imported again. It was already imported at "C:\Users\bernard\source\repos\icerpc-cshar
p\src\IceRpc.Templates\Directory.Build.props (3,5)". This is most likely a build authoring error. This subsequent import will be ignored. [C:\Users\bernard\source\repos\icerpc-csharp\src\IceRpc.Templates\IceRpc.Templates.csproj]
C:\Users\bernard\source\repos\icerpc-csharp\build\Common.props(4,5): warning MSB4011: "C:\Users\bernard\source\repos\icerpc-csharp\build\IceRpc.Version.props" cannot be imported again. It was already imported at "C:\Users\bernard\source\repos\icerpc-cshar
p\src\IceRpc.Templates\Directory.Build.props (3,5)". This is most likely a build authoring error. This subsequent import will be ignored. [C:\Users\bernard\source\repos\icerpc-csharp\src\IceRpc.Templates\IceRpc.Templates.csproj]
  Determining projects to restore...
  Restored C:\Users\bernard\source\repos\icerpc-csharp\src\IceRpc.Templates\IceRpc.Templates.csproj (in 48 ms).
C:\Users\bernard\source\repos\icerpc-csharp\build\Common.props(4,5): warning MSB4011: "C:\Users\bernard\source\repos\icerpc-csharp\build\IceRpc.Version.props" cannot be imported again. It was already imported at "C:\Users\bernard\source\repos\icerpc-cshar
p\src\IceRpc.Templates\Directory.Build.props (3,5)". This is most likely a build authoring error. This subsequent import will be ignored. [C:\Users\bernard\source\repos\icerpc-csharp\src\IceRpc.Templates\IceRpc.Templates.csproj]
  IceRpc.Templates -> C:\Users\bernard\source\repos\icerpc-csharp\src\IceRpc.Templates\bin\Any CPU\Debug\net7.0\IceRpc.Templates.dll
C:\Users\bernard\source\repos\icerpc-csharp\build\Common.props(4,5): warning MSB4011: "C:\Users\bernard\source\repos\icerpc-csharp\build\IceRpc.Version.props" cannot be imported again. It was already imported at "C:\Users\bernard\source\repos\icerpc-cshar
p\src\IceRpc.Templates\Directory.Build.props (3,5)". This is most likely a build authoring error. This subsequent import will be ignored. [C:\Users\bernard\source\repos\icerpc-csharp\src\IceRpc.Templates\IceRpc.Templates.csproj]
C:\Users\bernard\source\repos\icerpc-csharp\build\Common.props(4,5): warning MSB4011: "C:\Users\bernard\source\repos\icerpc-csharp\build\IceRpc.Version.props" cannot be imported again. It was already imported at "C:\Users\bernard\source\repos\icerpc-cshar
p\src\IceRpc.Templates\Directory.Build.props (3,5)". This is most likely a build authoring error. This subsequent import will be ignored. [C:\Users\bernard\source\repos\icerpc-csharp\src\IceRpc.Templates\IceRpc.Templates.csproj]
  Successfully created package 'C:\Users\bernard\source\repos\icerpc-csharp\src\IceRpc.Templates\bin\Any CPU\Debug\IceRpc.Templates.0.1.1.nupkg'
```

--- a/src/IceRpc.Templates/Directory.Build.props
+++ b/src/IceRpc.Templates/Directory.Build.props
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <Import Project="$(MSBuildThisFileDirectory)../../build/IceRpc.Version.props" />
     <Import Project="$(MSBuildThisFileDirectory)../../build/Common.props" />
     <ItemGroup>
         <GlobalAnalyzerConfigFiles Include="$(MSBuildThisFileDirectory)../../build/CodeAnalysis.Base.globalconfig" />

